index.js

```diff
@@ -378,11 +378,27 @@ function Argv (processArgs, cwd) {
   }
 
   var versionOpt = null
-  self.version = function (ver, opt, msg) {
-    versionOpt = opt || 'version'
-    usage.version(ver)
+  self.version = function (opt, msg, ver) {
+    var version = null
+    var versionMsg = null
+
+    if (arguments.length === 1) {
+      version = opt
+      versionOpt = 'version'
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else if (arguments.length === 2) {
+      versionOpt = opt
+      version = msg
+      versionMsg = usage.deferY18nLookup('Show version number')
+    } else {
```

Comment:
@bcoe @nexdrew What do you think? Should I integrate the default value for the version thing in this pull request so we can simplify things? If I don't, that would mean that we merge logic that is just there for covering the version parameter to be non-optional, once we put in the default value for the version, we wouldn't need that code anymore anyway.